packages/web-components/src/components/va-additional-info/va-additional-info.tsx

```diff
@@ -96,25 +100,52 @@ export class VaAdditionalInfo {
   }
 
   render() {
-    return (
-      <Host>
-        <a
-          role="button"
-          aria-controls="info"
-          aria-expanded={this.open ? 'true' : 'false'}
-          tabIndex={0}
-          onClick={this.toggleOpen.bind(this)}
-          onKeyDown={this.handleKeydown.bind(this)}
-        >
-          <div>
-            <span class="additional-info-title">{this.trigger}</span>
-            <i class="fa-angle-down" role="presentation" />
+    if (this.uswds) {
+      const infoClass = classnames({
+        'open': this.open,
+        'closed': !this.open
+      });
+      return (
+        <Host>
+          <a
+            role="button"
+            aria-controls="info"
+            aria-expanded={this.open ? 'true' : 'false'}
```

Comment:
If we're going to start using the `classnames` package, could it be used here for the aria-expanded too?

Reply:
yea I agree, I just thought to optimize it a little but I can revert back too